docs/root/overview.md

```diff
@@ -189,3 +189,9 @@ other formats (e.g. human, fortio). It can be very useful to always store the
 json output format, yet be able to easily get to one of the other output
 formats. It’s like having the cake and eating it too!
 
+## User-specified Nighthawk logging
+
+Users can use their own Nighthawk logging options when calling **ENVOY_LOG**.
```

Comment:
We may want to update the first sentence, since users don't really call ENVOY_LOG directly. It also isn't clear what we mean by "setting their own options". Let's try to write this from the user's perspective, how about something like this:

Users of Nighthawk can specify custom format and destination for all messages logged by Nighthawk. Nighthawk utilizes the Envoy's logging mechanism by performing all logging via the **ENVOY_LOG** macro. To customize this mechanism, users need to perform two steps:
...

Reply:
This sounds better to me indeed. Thanks for the suggestion!